composer/trainer/trainer.py

```diff
@@ -177,7 +209,25 @@ def __init__(
         save_compression: str = '',
 
         # Profiling
-        profiler: Optional[ProfilerHparams] = None,
+        profiling: bool = False,
+        prof_merged_trace_file: str = "merged_profiler_trace.json",
```

Comment:
See docstring comment above
```suggestion
        prof_merged_trace_file: Optional[str] = None,
```